store/sqlstore/post_store.go

```diff
@@ -882,9 +879,9 @@ func (s *SqlPostStore) GetPostsSince(options model.GetPostsSinceOptions, allowFr
 		WHERE
 		       UpdateAt > :Time AND ChannelId = :ChannelId
 		       LIMIT 1000)
-		(SELECT *` + replyCountQuery2 + ` FROM cte)
+		(SELECT cte.*, COALESCE(T.ReplyCount, 0) as ReplyCount FROM cte LEFT JOIN Threads as T ON T.PostId = (CASE WHEN cte.RootId IS NULL OR cte.RootId = '' THEN cte.Id ELSE cte.RootId END))
```

Comment:
@jespino this subquery used to run under SkipFetchThreads flag. Now it will run always. Is that what expected? Might cause slight degradation ...

Reply:
Good point, I'm not sure the reasoning behind that, I think including the replyCount there is going to be a bit less performant, I can change it.